caravel/static/default.css

```diff
@@ -168,4 +154,73 @@ form .thumbnail {
 .list-grp{
     display: inline-block;
     float: right;
+}
+
+/* Custom header via media query */
+@media (min-width: 992px){
+    .logo-brand-block{
```

Comment:
Cuz bootstrap cols don't work well with the col spacing required. Thats why there was weird amounts of whitespace.